gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -250,6 +250,14 @@ export default function NxTablePage() {
                           liveExample={NxTableFilterExample}
                           codeExamples={tableFilterExample}>
         An example with filter columns.
+        <p>
+          The first column has a basic filter input, the rows will be filtered
+          if any name contains the text provided in the input.
+        </p>
+        <p>
+          The second column has a filter input which provides a suggestion capability, the rows will be filtered
+          when the country is equal to the chosen suggestion.
```

Comment:
Actually they seem to filter based on substrings just like the other filter.  For instance "s" matches USA.

Reply:
Exactly: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/44e2eb852a34641822edbe5670ddf0439f11771c